safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -115,7 +115,7 @@ def __init__(self):
                 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 1.0,
                 7: 1.0, 8: 1.0, 9: 1.0, 10: 1.0
             }),
-            ('mmi_range', range(2, 10)),
+            ('mmi_range', range(4, 11)), # from MMI 4 to 10
```

Comment:
I think MMI < 4 is not likely to make any fatality. Also note that mmi < 6 is ignored in the current estimation of displaced number.